pypeit/scripts/setup_coadd2d.py

```diff
@@ -147,7 +147,7 @@ def main(args):
                                 else [f.name.split('-')[1].split('_')[0] for f in spec2d_files])
         if args.obj is not None:
             # Limit to the selected objects
-            _objects = [o for o in objects if o == args.obj]
+            _objects = [o for o in objects if o in args.obj]
```

Comment:
Can you say more about this change?  The original idea was that the object name should be visible via the spec2d file names, and the user should be able to set it exactly.  I appreciate there could be an issue where, e.g., human error led to a slightly changed the object name written to the header.  Is the idea to allow for slight deviations of the string as long as at least some fraction of it is common for all files?  And, if so, wouldn't you want to do `if args.obj in o` instead of the other way around?

Reply:
``args.obj`` is a list.  I got a strange error along the lines of ``here is everthing: ['obj1', 'obj2', 'JWST', 'obj3'], but I couldn't find ['JWST']``

Reply:
We can put this in a separate PR, if you want.

Reply:
DevSuite launched.